schemas/components/schemas/ClonedApp.yaml

```diff
@@ -0,0 +1,11 @@
+type: object
+additionalProperties: false
+required:
+  - result
+properties:
+  result:
+    type: boolean
+    description: >
+      Android specific cloned application detection. There are 2 values: 
+        • `true` - Presence of app cloners work detected (e.g. fully cloned application found or launch of it inside of a not main working profile detected).
+        • `false` - No signs of cloned application detected or the client is not Android.
```

Comment:
@nviktorov This is not a problem with your PR at all, it was like this before. But basically any time we have `description: >` (Yaml folds newlines into spaces) I think we should use `description: |` instead (newlines are kept, which allows the description to be rendered as Markdown in the documentation).

```suggestion
    description: |
      Android specific cloned application detection. There are 2 values:    
      * `true` - Presence of app cloners work detected (e.g. fully cloned application found or launch of it inside of a not main working profile
      * `false` - No signs of cloned application detected or the client is not Android.
```

![image](https://github.com/user-attachments/assets/4b5fb14c-d093-4f7a-ac24-f1a474ce1746)
vs 
![image](https://github.com/user-attachments/assets/1439a090-9e9a-42e5-a05e-7587bd4499ea)

If you want I can push a commit to your PR fixing this or I can make my own subsequent PR, let me know what you prefer. Thanks a lot!

cc @ilfa

Reply:
Fixed in the 3rd commit.